primeqa/components/retriever/dense.py

```diff
@@ -1,10 +1,13 @@
 from typing import List, Any
 from dataclasses import dataclass, field
 import json
+import os
```

Comment:
Formatting request. Can you push import statement for os under typing import? 
```
from typing import List, Any
import os
from dataclasses import dataclass, fields
import json
```

Reply:
Updated as suggested.